infra/README.md

```diff
@@ -1,5 +1,23 @@
 # symptomradar infra
 
+## Getting started
+
+Follow these instructions to set up your environment so that you can contribute to the development of the project.
+
+1. Install Terraform. On macOS, the simplest way is probably `brew install terraform`.
+1. Obtain secrets for your `.envrc` file (ask around in the team).
+1. Move to the `infra/` dir and run `terraform init` to set up Terraform.
+1. You're done!
+1. To check, you can run `terraform plan`. It should refresh a bunch of resources, and eventually tell you that "No changes. Infrastructure is up-to-date."
```

Comment:
Note: If you do this in between releases and there's changes in current `master`, there might be changes which might confuse the unfortunate user.

Reply:
Hmm, this shouldn't be the case if there's changes on `master` and they've been applied - but of course if someone is doing infra work on dev you might see some WIP.

I'll add a quick note!